service/builder.go

```diff
@@ -22,31 +22,25 @@ package service
 
 import "github.com/pkg/errors"
 
+// WithModules is a helper to create a service without any options
+func WithModules(modules ...ModuleCreateFunc) *Builder {
```

Comment:
nit: that looks a bit weird from a user perspective: `service.WithModules` returns a builder, not a service.